Silicon/TigerlakePchPkg/Library/GpioPlatformLib/GpioPlatformLib.inf

```diff
@@ -0,0 +1,41 @@
+## @file
+#
+#  Copyright (c) 2020, Intel Corporation. All rights reserved.<BR>
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+
+[Defines]
+  INF_VERSION                    = 0x00010005
+  BASE_NAME                      = GpioPlatformLib
+  FILE_GUID                      = E2A7C749-62AC-49F4-B4FA-26189C164E05
+  MODULE_TYPE                    = BASE
+  VERSION_STRING                 = 1.0
+  LIBRARY_CLASS                  = GpioPlatformLib
+
+#
+# The following information is for reference only and not required by the build tools.
+#
+#  VALID_ARCHITECTURES           = IA32 X64
+#
+
+[Sources]
+  GpioPlatformLib.c
+
+[Packages]
+  MdePkg/MdePkg.dec
+  BootloaderCommonPkg/BootloaderCommonPkg.dec
+  Silicon/TigerlakePchPkg/TigerlakePchPkg.dec
+  Platform/TigerlakeBoardPkg/TigerlakeBoardPkg.dec

```

Comment:
Platform Pkg is required?

Reply:
Without this, we cant access CDATA_GPIO_TAG.

Reply:
where we need access CDATA_GPIO_TAG? or could it be passed to the gpio library as parameter?
In general, PCH/silicon package should not depend on platform package.

Reply:
Ok, I can make that change.